apps/omg_watcher/lib/exit_validator/validator.ex

```diff
@@ -36,10 +33,7 @@ defmodule OMG.Watcher.ExitValidator.Validator do
   end
 
   defp exists?(utxo_exit) do
-    utxo_position = utxo_exit.utxo_pos
-    blknum = div(utxo_position, @block_offset)
-    txindex = utxo_position |> rem(@block_offset) |> div(@transaction_offset)
-    oindex = utxo_position - blknum * @block_offset - txindex * @transaction_offset
+    {:utxo_position, blknum, txindex, oindex} = OMG.API.Utxo.Position.decode(utxo_exit.utxo_pos)
```

Comment:
Please use explicit match `Utxo.position(blknum, txindex, oindex) = OMG.API.Utxo.Position.decode(utxo_exit.utxo_pos)` and require macro from `require OMG.API.Utxo`